src/Bicep.LangServer/Completions/BicepCompletionProvider.cs

```diff
@@ -509,6 +580,25 @@ private static CompletionItem CreateResourceTypeCompletion(ResourceTypeReference
                 .WithSortText(index.ToString("x8"));
         }
 
+        private static CompletionItem CreateModulePathCompletion(string name, string path, Range replacementRange, CompletionItemKind completionItemKind, CompletionPriority priority, bool adjustCursor)
+        {
+            // replace windows type path sep (\\<file>) with unix path sep (/file)
+            path = StringUtils.EscapeBicepString(path.Replace(@"\\", @"/"));
```

Comment:
Why would path contain a windows path separator if we're just working with URIs? Note that `\` is a valid file/dir character on unix.